lib/dice.py

```diff
@@ -0,0 +1,9 @@
+from random import randint
+
+    def roll(self, number, sides, modifier):
+        finalroll = 0
+        for x in range(0, number):
+            roll = (randint(1, sides))
+            finalroll = finalroll + roll
+        modifiedroll = finalroll + modifier
+        return modifiedroll
```

Comment:
you can compress these two lines into
`return modifiedroll + finalroll`

You should probably be using snake case for these too, ie `modified_roll`

Reply:
return modifiedroll+ finalroll would mean that the modifier was never added